# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-05-02 17:27+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: logic/check_folder_structure.py:157
msgid "zip.errors.invalid_structure.blocked_extension_found"
msgstr "The submitted zip file contains a file with a non-allowed extension."

#: logic/check_folder_structure.py:161 logic/check_folder_structure.py:213
msgid "zip.success"
msgstr "The submitted zip file succeeds in all checks."

#: logic/check_folder_structure.py:164
msgid "zip.errors.invalid_structure.obligated_extension_not_found"
msgstr ""
"The submitted zip file doesn't have any file with a certain file extension "
"that's obligated."

#: logic/check_folder_structure.py:191
msgid "zip.errors.invalid_structure.directory_not_defined"
msgstr "An obligated directory was not found in the submitted zip file."

#: logic/check_folder_structure.py:212
msgid "zip.errors.invalid_structure.directory_not_found_in_template"
msgstr ""
"There was a directory found in the submitted zip file, which was not asked "
"for."

#: models/docker.py:8
msgid "dockerimage.state.queued"
msgstr "Docker image is in the queue"

#: models/docker.py:9
msgid "dockerimage.state.building"
msgstr "Docker image is building"

#: models/docker.py:10
msgid "dockerimage.state.eady"
msgstr "Docker image is ready"

#: models/docker.py:11
msgid "dockerimage.state.error"
msgstr "Docker image failed to build"

#: models/submission.py:67
msgid "submission.state.queued"
msgstr "Queued"

#: models/submission.py:68
msgid "submission.state.running"
msgstr "Running"

#: models/submission.py:69
msgid "submission.state.success"
msgstr "Success"

#: models/submission.py:70
msgid "submission.state.failed"
msgstr "Failed"

#: models/submission.py:75
msgid "submission.error.blockedextension"
msgstr "The zip file contains a file with a non-allowed extension."

#: models/submission.py:76
msgid "submission.error.obligatedextensionnotfound"
msgstr ""
"The submitted zip file doesn't have any file with an obligated file "
"extension."

#: models/submission.py:77
msgid "submission.error.obligateddirectorynotfound"
msgstr "The submitted zip file doesn't have an obligated directory."

#: models/submission.py:78
msgid "submission.error.unaskeddirectory"
msgstr "The zip file contains a non allowed directory."

#: models/submission.py:81
msgid "submission.error.dockerimageerror"
msgstr "try again later."

#: models/submission.py:82
msgid "submission.error.timelimit"
msgstr "Timelimit exceeded."

#: models/submission.py:83
msgid "submission.error.memorylimit"
msgstr "Memorylimit exceeded."

#: models/submission.py:84
msgid "submission.error.runtimeerror"
msgstr "Crashed."

#: models/submission.py:85
msgid "submission.error.unknown"
msgstr "Unkown error."

#: serializers/checks_serializer.py:50 tests/test_project.py:494
msgid "project.error.structure_checks.already_existing"
msgstr "The structure check is already present in the project."

#: serializers/checks_serializer.py:66 tests/test_project.py:528
msgid "project.error.structure_checks.extension_blocked_and_obligated"
msgstr "An extension can't be blocked and obligated at the same time."

#: serializers/checks_serializer.py:103
msgid "extra_check.error.docker_image"
msgstr "The field 'docker_image' is required."

#: serializers/checks_serializer.py:106
msgid "extra_check.error.timeout"
msgstr "The field 'timeout' cannot be greater than 1000."

#: serializers/course_serializer.py:100 serializers/course_serializer.py:119
#: serializers/course_serializer.py:138 serializers/course_serializer.py:157
msgid "courses.error.context"
msgstr "The course is not supplied in the context."

#: serializers/course_serializer.py:106 tests/test_locale.py:28
#: tests/test_locale.py:38
msgid "courses.error.students.already_present"
msgstr "The student is already present in the course."

#: serializers/course_serializer.py:110 serializers/course_serializer.py:129
#: serializers/course_serializer.py:148 serializers/course_serializer.py:167
msgid "courses.error.past_course"
msgstr "The course is from a past year, thus cannot be manipulated."

#: serializers/course_serializer.py:125
msgid "courses.error.students.not_present"
msgstr "The student is not present in the course."

#: serializers/course_serializer.py:144
msgid "courses.error.teachers.already_present"
msgstr "The teacher is already present in the course."

#: serializers/course_serializer.py:163
msgid "courses.error.teachers.not_present"
msgstr "The teacher is not present in the course."

#: serializers/course_serializer.py:171
msgid "courses.error.teachers.last_teacher"
msgstr "The course must have at least one teacher."

#: serializers/course_serializer.py:116
msgid "courses.error.invitation_link"
msgstr "The invitation link is not unique, please try again."

#: serializers/docker_serializer.py:19
msgid "docker.errors.custom"
msgstr "User is not allowed to create public images"

#: serializers/group_serializer.py:47
msgid "group.errors.score_exceeds_max"
msgstr "The score exceeds the group's max score."

#: serializers/group_serializer.py:57 serializers/group_serializer.py:87
msgid "group.error.context"
msgstr "The group is not supplied in the context."

#: serializers/group_serializer.py:65 serializers/group_serializer.py:99
msgid "group.errors.locked"
msgstr "The group is currently locked."

#: serializers/group_serializer.py:69
msgid "group.errors.full"
msgstr "The group is already full."

#: serializers/group_serializer.py:73
msgid "group.errors.not_in_course"
msgstr "The student is not present in the related course."

#: serializers/group_serializer.py:77
msgid "group.errors.already_in_group"
msgstr "The student is already in the group."

#: serializers/group_serializer.py:95
msgid "group.errors.not_present"
msgstr "The student is currently not in the group."

#: serializers/project_serializer.py:46
msgid "project.errors.context"
msgstr "The project is not supplied in the context."

#: serializers/project_serializer.py:50
msgid "project.errors.start_date_in_past"
msgstr "The start date of the project lies in the past."

#: serializers/project_serializer.py:63
msgid "project.errors.deadline_before_start_date"
msgstr "The deadline of the project lies before the start date of the project."

#: serializers/project_serializer.py:108
msgid "project.errors.zip_structure"
msgstr "Error while parsing the provided zip."

#: serializers/project_serializer.py:141 tests/test_submission.py:333
msgid "project.error.submissions.past_project"
msgstr "The deadline of the project has already passed."

#: serializers/project_serializer.py:144 tests/test_submission.py:404
msgid "project.error.submissions.non_visible_project"
msgstr "The project is currently in a non-visible state."

#: serializers/project_serializer.py:147 tests/test_submission.py:434
msgid "project.error.submissions.archived_project"
msgstr "The project is archived."

#: tests/test_project.py:432
msgid "project.success.structure_check.add"
msgstr "A strucure check was successfully created for the project."

#: views/admin_view.py:29
msgid "admins.success.add"
msgstr "The admin was successfully added."

#: views/assistant_view.py:37 views/teacher_view.py:38
msgid "teachers.success.add"
msgstr "The teacher was successfully added."

#: views/assistant_view.py:70 views/teacher_view.py:71
msgid "teachers.success.destroy"
msgstr "The teacher was successfully destroyed."

#: views/course_view.py:133
msgid "courses.success.assistants.add"
msgstr "The assistant was successfully added to the course."

#: views/course_view.py:160
msgid "courses.success.assistants.remove"
msgstr "The assistant was successfully removed from the course."

#: views/course_view.py:222
msgid "courses.success.students.add"
msgstr "The student was successfully added to the course."

#: views/course_view.py:243
msgid "courses.success.students.remove"
msgstr "The student was successfully removed from the course."

#: views/course_view.py:288
msgid "courses.success.teachers.add"
msgstr "The teacher was successfully added to the course."

#: views/course_view.py:312
msgid "courses.success.teachers.remove"
msgstr "The teacher was successfully removed from the course."

#: views/group_view.py:75
msgid "group.success.students.add"
msgstr "The student was successfully added to the group."

#: views/group_view.py:95
msgid "group.success.students.remove"
msgstr "The student was successfully removed from the group."

#: views/group_view.py:114
msgid "group.success.submissions.add"
msgstr "The submission was successfully added to the group."

#: views/project_view.py:86
msgid "project.success.groups.created"
msgstr "A group was successfully created for the project."

#: views/project_view.py:156
msgid "project.success.extra_check.add"
msgstr "The extra check check was successfully added to the project."

#: views/student_view.py:36
msgid "students.success.add"
msgstr "The student was successfully added."

#: views/student_view.py:45
msgid "students.success.destroy"
msgstr "The student was successfully destroyed."
